schemas/remote_configuration_v2_template.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$id": "remote_configuration_v2_template",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "Config template files stored as object which contain list of conditional gathering rules and list of container logs",
+  "example": {
+    "conditional_gathering_rules": [
+      "conditional_gathering_rules/*.json",
+      "conditional_gathering_rules/rule1.json"
```

I am sorry, this example suggests that it is OK to include the same file twice. As I explained in in another comment, it is undesirable.

We can either keep the example short and simple (I assume that the actual value will look like this in the vast majority of cases):

```suggestion
      "conditional_gathering_rules/*.json"
```

... or make up something more elaborate:

```suggestion
      "conditional_gathering_rules/Alertmanager*.json",
      "conditional_gathering_rules/Prometheus*.json"
```